fix(fromOpenApi): remove extra whitespace from content type

Kiskae opened the pull request:
Fixes #61

kettanaito commented on src/open-api/utils/open-api-utils.ts:
```diff
@@ -263,6 +257,14 @@ export function toBody(
   return null
 }
 
+function getAcceptedContentTypes(request: Request): string[] {
```
Love this!

kettanaito reviewed (approved):
Thank you for jumping on this so fast, @Kiskae! The changes look great. The only thing that's missing is an automated test for this. 

Feel free to explore and add one, otherwise I will try to add it later next week. Thank you!

weyert commented:
@Kiskae Thanks for taking care of this. Never thought of the whitespaces 🤗 
Personally, barely ever pass multiple content-types to `Accept`

weyert reviewed (approved):
LGTM

Kiskae commented:
> Thank you for jumping on this so fast, @Kiskae! The changes look great. The only thing that's missing is an automated test for this.
> 
> Feel free to explore and add one, otherwise I will try to add it later next week. Thank you!

Added some tests. After looking at the RFC I realized that there are more complex accept headers that aren't being handled, but solving that probably involves adding dependencies. So I've just added some skipped tests as a reminder that they exist.

Also made it so the automated tests in src actually get executed :P

kettanaito commented:
This looks fantastic! Thank you for adding the tests, @Kiskae!

I've polished the things a bit, mostly around test names and changing the `getAcceptedContentTypes` to accept just the `Headers` instance (it never needed anything else from the request). Made it easier to test too.

kettanaito commented:
## Released: v0.3.1 🎉

This has been released in v0.3.1!

- 📄 [**Release notes**](https://github.com/mswjs/source/releases/tag/v0.3.1)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/source/v/0.3.1)

Make sure to always update to the latest version (`npm i @mswjs/source@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
